generators/PHPythia8/PHPythia8.C

```diff
@@ -249,12 +279,13 @@ int PHPythia8::process_event(PHCompositeNode *topNode)
 
   // save statistics
   if (_integral_node)
-  {
-    _integral_node->set_N_Generator_Accepted_Event(_pythia->info.nAccepted());
-    _integral_node->set_N_Processed_Event(_eventcount);
-    _integral_node->set_Sum_Of_Weight(_pythia->info.weightSum());
-    _integral_node->set_Integrated_Lumi(_pythia->info.nAccepted() / (_pythia->info.sigmaGen() * 1e9));
-  }
+    {
+      _integral_node->set_N_Generator_Accepted_Event(_pythia->info.nAccepted());
+      _integral_node->set_N_Processed_Event(_eventcount);
+      _integral_node->set_Sum_Of_Weight(_pythia->info.weightSum());
+      _integral_node->set_Integrated_Lumi(_pythia->info.nAccepted() / (_pythia->info.sigmaGen() * 1e9));
```

Comment:
Are these variables correct in case of both `_legacy_vertexgenerator` = true and false?

Reply:
Yes, they work for both cases.

Reply:
I wonder how they work.  I looked at only the diffs, but I understand `_pythia` points to `ppGen` or `pnGen`, and thus only one of them is  here.  But do we need both (i.e. sum) of them?

Reply:
Dear Kenichi. yes, `_pythia` point to either `ppGen` or `pnGen`. As these values are incremented event-wise, they both should have contribution towards the total events' integrated luminosity. Does that make sense?

Reply:
> they both should have contribution towards the total events' integrated luminosity.
>
I understand so.  Then the present code is not sufficient.  Instead the sum like `ppGen->info.nAccepted() + pnGen->info.nAccepted()` should be used.

Reply:
Dear Kenichi,
Thank you for the detail look. yes, it does seems like we need to set the integral node separately in case of `_legacy_vertexgenerator = true` by adding both from `ppGen` and `pnGen`